presto-product-tests/bin/product-tests-suite-3.sh

```diff
@@ -22,5 +22,12 @@ presto-product-tests-launcher/bin/run-launcher test run \
     -- -g storage_formats,cli,hdfs_impersonation,authorization -x iceberg,"${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
     || suite_exit_code=1
 
+# run very selective set of tests on singlenode-kerberos-hdfs-impersonation-with-data-protection so we do not add
+# a lot to runtime
+presto-product-tests-launcher/bin/run-launcher test run \
+    --environment singlenode-kerberos-hdfs-impersonation-with-data-protection \
+    -- -t TestHiveStorageFormats.testOrcTableCreatedInPresto,TestGrantRevoke.testGrantRevoke -x "${DISTRO_SKIP_GROUP}" -e "${DISTRO_SKIP_TEST}" \
```

Comment:
Why `TestGrantRevoke` at all?

Reply:
Not a good answer here. I think we can skip that. I left it here as representant of authentication group, but I agree this is not really relevant to data-transfer-protection. I will add one or two more which actually read data to make probability that those will be excluded via DISTRO_SKIP* variables.